blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineUpdateRequest.java

```diff
@@ -55,6 +55,7 @@ public BluePipeline update(BluePipeline pipeline) throws IOException {
                 folder.scheduleBuild(new Cause.UserIdCause());
             }
         }
+        item.save();
```

Comment:
good point, when I was debugging (and by debuggin I mean print statements with swears in them) I never saw this one called. only put it in there for completeness.